qiita_db/reference.py

```diff
@@ -23,6 +79,7 @@ class Reference(QiitaObject):
     sequence_fp
     taxonomy_fp
     tree_fp
+    sortmerna_indexed_db
```

Comment:
@adamrp, taxonomy and tree are still valid for a focused protein database, though it does suggest that we need to rethink the concept of a sequence reference here as we need to support multi locus dbs. Note also that the property on like 249 will raise not warn